packages/api-server/api_server/models/tortoise_models/tasks.py

```diff
@@ -30,6 +30,14 @@ class TaskState(Model):
     unix_millis_warn_time = DatetimeField(null=True, index=True)
     pickup = CharField(255, null=True, index=True)
     destination = CharField(255, null=True, index=True)
+    labels = ReverseRelation["TaskLabel"]
+
+
+class TaskLabel(Model):
+    state = ForeignKeyField("models.TaskState", null=True, related_name="labels")
```

Comment:
Because we do not have a canonical "Task" object, this foreign key is causing issues with task requests and task states. I would say better to remove the foreign key constraint and just use task booking id.

Reply:
Resolved as discussed